crates/re_data_store/benches/arrow2.rs

```diff
@@ -113,7 +113,7 @@ fn erased_clone(c: &mut Criterion) {
             .sum::<u64>();
         let expected_total_size_bytes = data.total_size_bytes();
         assert!(
-            total_size_bytes >= expected_total_size_bytes,
```

Comment:
Apparently the size calculation is one byte off -- this bench doesn't run on CI. Whatever.

Reply:
you should probably add a comment though - the new code is quite confusing to the reader